.gitignore

```diff
@@ -8,3 +8,5 @@ docs
 
 .idea/
 .vscode/
+
+.coveralls.yml
```

Comment:
why ignore the coveralls file, doesn't it contain config?

Reply:
It's used to run coveralls locally and contains the repo token, which should not be made public